### PR TITLE
Rename CBL0033J to CBL0003J

### LINKED ISSUE #198: no CBL0033J
Hi,
In pdf 1, they speak about CBL0033J in 3. of 10.4 Lab.
But, in the mainframe, I can not find any. Can you try and tell me, please ?

### PR DESCRIPTION
**Pull request by tanto259**
Signed-off-by: Hartanto Ario Widjaya <tanto259@users.noreply.github.com>

This would fix #198. It appears #153 is not merged into the mainframe system, thus resulting in confusion with the PDF content.

Personally, I would prefer that the new CBL0003.cbl be renamed to something else or adopt the CBL0033 naming (8af11a09e2218c03901ebf877b40fbceb73e02b6) to prevent confusion with existing tutorials and PDF which are already on circulation (such as Coursera, Pluralsight and the IBM Learning Platform).

### CODE REVIEW

**Comment by MikeBauerCA** [comment]
Thanks for contributing @tanto259. If @paulnewt just added that CBL0033J (https://github.com/openmainframeproject/cobol-programming-course/blob/master/COBOL%20Programming%20Course%20%231%20-%20Getting%20Started/Labs/jcl/CBL0033J.jcl) member on the mainframe, would that resolve this issue? Tagging @klausmelcher who contributed #153 as well.

**Comment by MikeBauerCA** [comment]
Makes sense to me. We need to enable the system to support new content and old content simultaneously to accommodate all users. I'll wait for @paulnewt's comment before approving so he is aware of the needed update.

**Comment by paulnewt** [comment]
ZOS.PUBLIC.CBL member CBL0003 now exists.  ZOS.PUBLIC.JCL member CBL0033J now exists.  Member CBL0003J and CBL0033J compile newly added source CBL0003

**Comment by tanto259** [comment]
Thanks for the upload @paulnewt !

But I think there's some issues here:

1. Originally, before #153, CBL0003J is a JCL which is intentionally designed to have a bug, in which it executes CBL0001 but with an incorrect DD name
2. Klaus then submitted #153, in which he added a COBOL program which he called CBL0003 and the corresponding JCL CBL0003J, which is designed to explain subroutine calls. Since CBL0003J already existed previously, he renamed the old one (which is buggy) to CBL0033J.
3. However, this causes some issues since the Lab files on both Coursera and IBM Learning Platform still references the buggy JCL as CBL0003J instead of its new name, CBL0033J. Thus, in order to maintain compatibility with prior docs out there, I open this pull request.

Basically what this pull request did is bring back the renamed JCL to CBL0003J and changes Klaus' COBOL and JCL to adopt the CBL0033 naming. But the update basically removed the buggy JCL and put an exact copy of CBL0001 as CBL0003, so Klaus' code is still not on the system.

**Comment by MikeBauerCA** [comment]
@tanto259 @paulnewt the confusion comes from CBL0003J.jcl executing CBL0001.cobol and CBL0033J.jcl executing CBL0003.cobol. We don't want to change the former because of backwards compatibility on existing tutorials in circulation that are somewhat out of date. But we can change the latter without much impact.

Going forward we should avoid renaming existing programs for backwards compatibility reasons.

@tanto259 can you review what is on the system and work with @paulnewt if an update is needed. The PR looks good to merge once we verify the system matches.

To summarize:
CBL0003J compiles CBL0001
CBL0033J compiles CBL0033
CBL0003 should not exist to avoid confusion

**Comment by tanto259** [comment]
Sure. @paulnewt, here are the changes needed to the system:

- rename CBL0003.cbl to CBL0033.cbl in ZOS.PUBLIC.CBL
- move CBL0003J and CBL0033J from ZOS.PUBLIC.ADHOC.JCL to ZOS.PUBLIC.JCL and overwrite the old one there

Those should correspond to the changes made in this pull request. At the end of the day, CBL0003.cbl should no longer exist on the system.

**Comment by tanto259** [comment]
@MikeBauerCA I have talked to Paul, and he has merged the required changes to the system.

ZOS.PUBLIC.CBL(CBL0033) now exists, CBL0003 is no longer there, and ZOS.PUBLIC.JCL(CBL0003J and CBL0033J) updated.

**Comment by MikeBauerCA** [comment]
@tanto259 thanks, if you can resolve the merge conflicts in this PR, I will accept and merge.

**Review by MikeBauerCA — APPROVED** [review]
